src/app/settings/settings.component.ts

```diff
@@ -86,10 +90,14 @@ export class SettingsComponent extends RouterEnter {
     this.newRelic.actionText('Open T&C link');
     window.open(this.termsUrl, '_system');
   }
-
   switchProgram() {
-    this.newRelic.actionText('browse to program switcher');
-    this.router.navigate(['/switcher']);
+    if (this.settingService.LtiReturnUrl) {
```

Comment:
We shouldn't store this in the settingService, it is a information that should be globally accessible, so that we can store it in local storage, either as part of the "me" object, or make it a stand along key/value pair. If we are going to make it a stand alone key/value pair, don't simply use storage.get(), create functions in storage service for get/set like what we do for getUser/setUser and getConfig/setConfig.
I recommend to just store it as part of the "me" object